src/App.jsx

```diff
@@ -45,17 +55,17 @@ export function App() {
 			/** Finally, we update our React state. */
 			setData(nextData);
 		});
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [listToken]);
```

Comment:
When I was testing this I noticed it allowed me to go to the home screen when I had a token in local storage but the app should redirect me to /list when I have a token in local storage. I added `navigateTo` to the dependency array and it gave me the behavior I expected!

Reply:
Hi Lindsey, I think I am confused about the acceptance criteria and expected behavior of the app and navigation. I may have also received some conflicting information and was hoping you could clarify. 

Initially, Zenzi requested some changes because "The issue is the inability to navigate to add-item or home afterwards." Afterwards meaning after the listToken as been generated. I made the changes you see (removing navigateTo from the dependency & disabling eslint next line) in order to satisfy this requirement. I was able to navigate from the list page, back to Home and to Add Item using the nav links, but the Home route still redirected to the list view if manually entered in the URL.

I did go ahead and move the useEffect with navigateTo('/list') into the Home component and as you expected, I am no longer able to navigate to the Home route using the navLinks but I can still navigate to the Add Item route. I can go ahead and push these changes up, but it conflicts with Zenzi's requested changes. 

Please advise?